packages/vite/src/node/plugins/html.ts

```diff
@@ -938,6 +940,8 @@ export interface IndexHtmlTransformContext {
   bundle?: OutputBundle
   chunk?: OutputChunk
   originalUrl?: string
+  req?: Connect.IncomingMessage
```

Comment:
Since `transformIndexHtml` is also called during dev server warmup, when there is no `req` / `res` to pass through, is this something which warrants additional documentation or comments?

https://github.com/vitejs/vite/blob/17fb5ee41f96e3e4aefc6d6258436676de048686/packages/vite/src/node/server/warmup.ts#L34-L40